doc/pages/components/reveal.html

```diff
@@ -124,7 +124,8 @@ <h3 class="subheader">Modal dialogs, or pop-up windows, are handy for prototypin
 
 ## Accessibility
 
-<p class="panel">This component is not yet fully accessible. While it is usable via the keyboard, it has to be checked if additional ARIA attributes can enhance the components accessibility.</p>
+<p class="panel">This component is not yet fully accessible. While it is usable via the keyboard, it has to be checked if additional (ARIA) attributes can enhance the components accessibility. The examples above show how to implement accessible modals using some of those attributes. 
+<br />Please note that you should either add the tabindex- or the href-attribute to anchors to make them fully utilizable by keyboards.</p>
```

Comment:
I would suggest writing out more, rather than less, about how to make links and buttons and modals accessible, and make this a fully fledged section in the page, outside of the `.panel`. 

Basically:

- `role="dialog"`
- `<button>` (or `role="button"`)
- `aria-label` or `aria-labelledby`
- `aria-describedby`
- `aria-hidden`